.github/workflows/CITest.yml

```diff
@@ -0,0 +1,21 @@
+name: CI of ide-mirrors
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Detect CRLF line endings
+        id: detect
+        run: find . -type f | xargs file | grep CRLF
+        continue-on-error: true
+      - name: Check if ext-mappings present in every subfolder
+        id: check
+        run: find . -maxdepth 1 -type d '!' -exec test -e "{}/ext-mappings" ';' -print
```

Comment:
IMHO this command will never fail. It will print the directories that do not contain `ext-mappings` folder and always exit with success (`0`). Did you test that it fails in such case and I am missing something? Dont this command always find the `.git` folder and if it could exit with failure on match it would always fail?

Reply:
@arexhepi7 thanks for the rework. You found a fix for this. 👍 I set to resolved.